web_src/js/features/repo-legacy.js

```diff
@@ -84,18 +84,17 @@ export function initRepoCommentForm() {
     $(`.${selector}`).dropdown('setting', 'onHide', () => {
       hasUpdateAction = $listMenu.data('action') === 'update'; // Update the var
       if (hasUpdateAction) {
-        const promises = [];
-        Object.keys(items).forEach((elementId) => {
-          const item = items[elementId];
-          const promise = updateIssuesMeta(
-            item['update-url'],
-            item.action,
-            item['issue-id'],
-            elementId,
-          );
-          promises.push(promise);
-        });
-        Promise.all(promises).then(() => window.location.reload());
+        (async function() {
```

Comment:
```suggestion
        // TODO: Add batch functionality and make this 1 network request.
        (async function() {
```
Let's add a simple TODO, so it won't be forgot that it can be improved.